src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -207,6 +207,13 @@ private native void nativeInit() /*-{
             }, i[r].l = 1 * new Date();
             a = s.createElement(o), m = s.getElementsByTagName(o)[0];
             a.async = 1;
+            // If current protocol is "file" we are on mobile
+            // So we explicitly specify https protocol for the script's source link
+            if ($wnd.location.protocol === "file:") {
+            	// CHECKSTYLE_OFF
+                g = "https:" + g;
+                // CHECKSTYLE_ON
+            }
             a.src = g;
             m.parentNode.insertBefore(a, m)
         })($wnd, $doc, 'script', '//www.google-analytics.com/analytics.js',
```

Comment:
The only reason is to keep the protocol http/https depending on user's current protocol. If that's not important to you then I'll update it with https always, the code would be much cleaner also.